### PR TITLE
Thin results OVAL steps in user manual

### PR DESCRIPTION
**Pull request by mpreisler**
Better instructions for OVAL thin results in user manual.

The main improvement is the OVAL directives example.

### CODE REVIEW

File: docs/manual/manual.adoc [hunk 1]
```diff
@@ -331,12 +331,44 @@ file containing exported variables from the XCCDF file, and
...
+---------------------------------------------------------------------------------------------------
+
+Usage of OVAL directives file when scanning OVAL component from a Source DataStream:
+
 ---------------------------------------------------------------------------------------------------
-$ oscap oval eval --directives directives.xml --datastream-id ds.xml --oval-id xccdf.xml --results oval-results.xml scap-ds.xml
+$ oscap oval eval --directives directives.xml --without-syschar --datastream-id ds.xml --oval-id oval.xml --results oval-results.xml scap-ds.xml
 ---------------------------------------------------------------------------------------------------
 
 OVAL results file contains, by default, exported system characteristics.
```
Comment (jan-cerny) [inline docs/manual/manual.adoc]: I would move this paragraph before the first usage of `--without-syschar`.

File: docs/manual/manual.adoc [hunk 2]
```diff
@@ -331,12 +331,44 @@ file containing exported variables from the XCCDF file, and
...
 
+Example of an OVAL directive file which enables thin results instead of
+full results:
+
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<oval_directives xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:oval="http://oval.mitre.org/XMLSchema/oval-common-5" xmlns:oval-res="http://oval.mitre.org/XMLSchema/oval-results-5" xmlns="http://oval.mitre.org/XMLSchema/oval-directives-5" xsi:schemaLocation="http://oval.mitre.org/XMLSchema/oval-results-5 oval-results-schema.xsd http://oval.mitre.org/XMLSchema/oval-common-5 oval-common-schema.xsd http://oval.mitre.org/XMLSchema/oval-directives-5 oval-directives-schema.xsd">
+  <generator>
+    <oval:product_name>OpenSCAP</oval:product_name>
+    <oval:schema_version>5.8</oval:schema_version> <!-- make sure the OVAL version matches your input -->
```
Comment (jan-cerny) [inline docs/manual/manual.adoc]: Why 5.8?

File: docs/manual/manual.adoc [hunk 3]
```diff
@@ -331,12 +331,44 @@ file containing exported variables from the XCCDF file, and
...
+
 ---------------------------------------------------------------------------------------------------
-$ oscap oval eval --directives directives.xml --datastream-id ds.xml --oval-id xccdf.xml --results oval-results.xml scap-ds.xml
+$ oscap oval eval --directives directives.xml --without-syschar --datastream-id ds.xml --oval-id oval.xml --results oval-results.xml scap-ds.xml
 ---------------------------------------------------------------------------------------------------
 
 OVAL results file contains, by default, exported system characteristics.
 OpenSCAP provides *--without-syschar* option to change this behavior.
+If your use-case require thin OVAL results you most likely also want
+to omit system characteristics.
```
Comment (yuumasato) [inline docs/manual/manual.adoc]: The results of `directives.xml` with `content=thin` is not influenced by `--without-syschar` option `collected_objects`.

File: docs/manual/manual.adoc (hunk 2 above)
Comment (mpreisler) [inline docs/manual/manual.adoc]: No reason, I had to choose something.

File: docs/manual/manual.adoc (hunk 3 above)
Comment (mpreisler) [inline docs/manual/manual.adoc]: Are you sure? I tried it and if I don't use `--without-syschar` I still get `collected-objects` in the system_characteristics element.

File: docs/manual/manual.adoc (hunk 1 above)
Comment (mpreisler) [inline docs/manual/manual.adoc]: Fixed

**Pull request merged**